StudyRingMeteor/client/forum/SubForum.js

```diff
@@ -1,2 +1,40 @@
 import './MainForum.html';
 import './SubForum.html';
+import { Rings } from "../../collections/rings/rings.js";
+
+Template.ringDiscussion.onCreated(function(){
```

Comment:
Since the name has changed to "RingDiscussion", you need to change everywhere this name is used to have a capital R.